launcher/ui/pages/global/MinecraftPage.ui

```diff
@@ -214,21 +214,55 @@
          <property name="title">
           <string>Native library workarounds</string>
          </property>
-         <layout class="QVBoxLayout" name="verticalLayout_11">
-          <item>
+         <layout class="QGridLayout" name="gridLayout">
+          <item row="0" column="0">
            <widget class="QCheckBox" name="useNativeGLFWCheck">
             <property name="text">
              <string>Use system installation of &amp;GLFW</string>
             </property>
            </widget>
           </item>
-          <item>
+          <item row="1" column="0">
+           <widget class="QLabel" name="labelGLFWPath">
+            <property name="text">
+             <string>&amp;GLFW library path</string>
+            </property>
+            <property name="buddy">
+             <cstring>lineEditGLFWPath</cstring>
+            </property>
+           </widget>
+          </item>
+          <item row="2" column="0">
            <widget class="QCheckBox" name="useNativeOpenALCheck">
             <property name="text">
              <string>Use system installation of &amp;OpenAL</string>
             </property>
            </widget>
           </item>
+          <item row="3" column="0">
+           <widget class="QLabel" name="labelOpenALPath">
+            <property name="text">
+             <string>&amp;OpenAL library path</string>
+            </property>
+            <property name="buddy">
+             <cstring>lineEditOpenALPath</cstring>
+            </property>
+           </widget>
+          </item>
+          <item row="1" column="1">
+           <widget class="QLineEdit" name="lineEditGLFWPath">
+            <property name="enabled">
+             <bool>false</bool>
+            </property>
+           </widget>
+          </item>
+          <item row="3" column="1">
+           <widget class="QLineEdit" name="lineEditOpenALPath">
+            <property name="enabled">
+             <bool>false</bool>
+            </property>
+           </widget>
+          </item>
```

Comment:
Is it possible to have these enter a "Can't find the {name} library automatically! Does it exist?" when it can't be found by Prism? 

Having the text automatically filled out so that the user doesn't have to type it out would be nice too, right now it's just *shown* and can't be edited as-is. Though then auto updating it might be a bit of a pain? Can easily disregard this concern.

Reply:
We use placeholder texts in other places as well for pre-set/auto-detected value (check the APIs page)

It's probably gonna auto-detect the right libraries 99% of the time, so we are fine here.